### ISSUE TITLE
LibreSign should generate an activity and notification email when a document is signed

### ISSUE THREAD

**Issue by DazeEnd** [LibreSign/libresign#2618]
**Is your feature request related to a problem? Please describe.**
If user "Admin" requests a signature from user "Signer_1", there is currently no notification sent to Admin when the document is signed. Admin _might_ receive an activity email if they have turned on notifications for the "A file or folder has been changed" notification, but if that notification is turned OFF, then Admin will need to keep checking the file to see if it has been signed.

**Describe the solution you'd like**
When a document is signed, I would like for LibreSign to generate an activity email that is sent to the person who requested the signature. I would expect that this notification could be turned off in Settings > Personal > Notifications like other notifications.

**Describe alternatives you've considered**
Alternately, an email could be sent to the person who requested the signature immediately after the document is signed. (Similar to email that is immediately sent to a signer when his signature is requested.)

**Additional context**
Mockup of new Settings option:
![Screenshot from 2024-03-28 16-24-27](https://github.com/LibreSign/libresign/assets/2699247/28aa3ce4-f804-4fc6-a1c9-d8f1c6f710a8)

Mockup of new notification email:
![Screenshot from 2024-03-28 16-51-58](https://github.com/LibreSign/libresign/assets/2699247/2ad81484-ef25-439a-8d65-41fdab1b28f1)

**Comment by vitormattos** [comment]
This idea close
- #2531

**Comment by DazeEnd** [comment]
Yeah, it looks like this is a duplicate of #2531. Sorry about that. I didn’t notice #2531 until you pointed it out.

**Comment by vitormattos** [comment]
Hi @DazeEnd check the newest release of LibreSign. We implemented this.

Thanks by your help to LibreSign project, it's very important to us.
